bindings/python/eblob_python.cpp

```diff
@@ -52,6 +53,29 @@ static void eblob_extract_id(const struct eblob_id &e, struct eblob_key &id)
 	eblob_extract_arr(e.id, id.id, &len);
 }
 
+/**
+ * __eblob_read_ll() - interruption-safe wrapper for pread(2)
+ */
+static int __eblob_read_ll(int fd, char *data, size_t size, off_t offset)
```

Comment:
>Why did you copy-paste this whole function from one already present in eblob code?

Saying the truth I don't know whether it will be a good idea to move __eblob_read_ll to eblob's public interface. And I am not sure about necessity of this wrapper (eblob_read_ll).
I can use only single pread without any wrappers here.